src/pages/plan/pdf/PDFSave.tsx

```
@@ -3,12 +3,19 @@ import PDFTemplate from './PDFTemplate';
 import { schedules, travelers } from './data/example';
```

ESLint

[error] 'schedules' is defined but never used.

(@typescript-eslint/no-unused-vars)